### PR TITLE
Update to use Heaviside 2.0

### PR DESCRIPTION
**Pull request by derek-pryor**
Updated heaviside submodule reference and library code to use the new Heaviside 2.0 release

### CODE REVIEW

File: cloud_formation/configs/activities.py [hunk 1]
```diff
@@ -230,5 +230,6 @@ def delete_sfns(session, domain):
     sfn.delete(session, names.query_deletes)
     sfn.delete(session, names.ingest_queue_populate)
     sfn.delete(session, names.ingest_queue_upload)
+    sfn.delete(session, names.volumetric_ingest_queue_upload)
```
Comment (movestill) [inline cloud_formation/configs/activities.py]: Thanks, I remember noticing that wasn't happening but forgot to add it in!

**Comment by movestill** [comment]
Nevermind, I see the submodule update. Didn't see it under the files tab.

File: cloud_formation/configs/idindexing.py [hunk 2]
```diff
@@ -153,32 +166,17 @@ def pre_init(bosslet_config):
 
 def post_init(bosslet_config):
     """Create step functions."""
-    names = bosslet_config.names
+    role = 'StatesExecutionRole-us-east-1 '
```
Comment (movestill) [inline cloud_formation/configs/idindexing.py]: Will the role eventually be configurable?

Comment (derek-pryor) [reply cloud_formation/configs/idindexing.py]: Configurable? It already is...
When Step Functions were first introduced they had not update IAM to allow use to create roles for Step Functions, and so we've continued to use the default role created by AWS even though we can now create custom IAM Step Function roles.

**Pull request merged**